hello/patch/README.md

```diff
@@ -0,0 +1,9 @@
+# Patch Workflow Sample
+This sample will demonstrate how to version a workflow. The sample workflow will execute a activity and fire a timer. 
+
+- First run the v1 workflow
```

Comment:
You'll have to give people commands to run things

Reply:
now its straight-forward with cmd args